frontend/demo/component/auto-grid/react/auto-grid-selection.tsx

```diff
@@ -0,0 +1,32 @@
+import { reactExample } from 'Frontend/demo/react-example'; // hidden-source-line
+import { autoGridHostStyles } from './auto-grid-host-styles'; // hidden-source-line
+import React, {
+  useState
+} from 'react';
+import { AutoGrid } from '@hilla/react-grid';
+import { ProductService } from 'Frontend/generated/endpoints';
+import ProductModel from 'Frontend/generated/com/vaadin/demo/fusion/crud/ProductModel';
+import Person
+  from "Frontend/generated/com/vaadin/demo/domain/Person";
+import Product
+  from "Frontend/generated/com/vaadin/demo/fusion/crud/Product";
```

Comment:
```suggestion
import Product
  from "Frontend/generated/com/vaadin/demo/fusion/crud/Product";
```

Unused import. Could also run prettier over the newly added examples once.